packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -226,7 +248,13 @@ export class LiveShareHost {
   public setFluidContainerId(containerId: string): Promise<liveShare.IFluidContainerInfo> {
     ensureSupported();
     return new Promise<liveShare.IFluidContainerInfo>((resolve) => {
-      resolve(sendAndHandleSdkError('interactive.setFluidContainerId', containerId));
+      resolve(
```

Comment:
@[AE-MS](https://github.com/AE-MS) AE-MS [on Nov 28, 2023](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2108/files#discussion_r1408600340)
This is probably a bigger question than what should be handled in this PR, so it absolutely should not block you from checking in, but should the message being passed and the ApiName constant being used just be the same? In most (all?) cases, they're literally the same string. And representing them twice opens the possibility of unintentional divergence.

@[KangxuanYe](https://github.com/KangxuanYe) KangxuanYe [2 hours ago](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2108/files#discussion_r1440941709)
I have thought of this question and my answer to this is: Ideally, yes. That is to say the value of funcName as a parameter in SDK request should be EXACTLY the same as value of ApiVersionTag and developers don't have to write the same string twice in the future.

However, for now, due to some constraints, i.e. tech debt, openLink special scenarios, we cannot make it. But what I will suggest to do after this project is to use ApiName as the value of funcName to make sure that API keeps the funcName to be exactly the same as apiVersionTag so that in the future, we could remove apiVersionTag and use funcName for telemetry (at that moment the funcNames are API names).

Does that make sense to you? @AE-MS
(@TrevorJoelHarris tagged you for my suggestion in the future)

Reply:
It's a tricky question. Most functions (and possibly all *new* functions should probably be the same string value. We have back compat scenarios though (tasks vs. dialog for example) where we want different telemetry points for the same func values very explicitly.

I don't think aaron is wrong that having separate string values here opens us up to potentially having different values we don't mean to have, but at least in my head I'd rather have too *much* detail that we then have to combine again in the telemetry layer ("these two tags are actually the same call") than not enough detail ("uh oh, this single tag is actually two different function calls") so erring on this side seems okay to me. Just my vote though!